feat: readd the tabs for installation (#766)
shopware/docs: products/paas/repository.md

|------------|-------------------|-------------------------------------------------------------------------|
| `origin`   | Project Code      | This remote contains all your project specific source code              |
| `platform` | PaaS Environment  | Changes pushed to this remote will be synced with your PaaS environment |

## Migrating from old template to the new template

If you used already the Shopware PaaS template from <https://github.com/shopware/paas>, please follow this guide to [migrate to the new structure](../../guides/installation/flex.md#how-to-migrate-from-production-template-to-symfony-flex).

Following tasks has to be done additionally to the flex migration:

- The root `.platform.app.yml` has been moved to `.platform/applications.yaml`
- The following services has been renamed:
    - `queuerabbit` to `rabbitmq`
    - `searchelastic` to `opensearch`

As the services has been renamed a complete new service will be created. Here are three possible options available:

> - Renaming them back again
> - Starting with a new service and re-index Elasticsearch
> - [Doing the Transitional upgrade with two services for some time parallel](https://docs.platform.sh/add-services/opensearch.html#upgrading)

Reviewer: ```suggestion
* Rename the services back again
* Start with a new service and re-index Elasticsearch
* [Perform the transitional upgrade of two services in parallel for some time](https://docs.platform.sh/add-services/opensearch.html#upgrading)
```
